s3cmd

```diff
@@ -2755,6 +2755,8 @@ def main():
     optparser.add_option(      "--cache-file", dest="cache_file", action="store", default="",  metavar="FILE", help="Cache FILE containing local source MD5 values")
     optparser.add_option("-q", "--quiet", dest="quiet", action="store_true", default=False, help="Silence output on stdout")
     optparser.add_option(      "--ca-certs", dest="ca_certs_file", action="store", default=None, help="Path to SSL CA certificate FILE (instead of system default)")
+    optparser.add_option(      "--ssl-cert", dest="ssl_cert_file", action="store", default=None, help="Path to client SSL certificate FILE")
+    optparser.add_option(      "--ssl-key", dest="ssl_key_file", action="store", default=None, help="Path to client SSL certificate key FILE")
```

Comment:
See previous comment about the option names

Reply:
No, names should not be made long. But in the description of the parameters I will make changes
For cURL, they are simply called "key" and "cert".